main/index.js

```diff
@@ -86,7 +86,8 @@ const ctx = {
   saveSaturnModuleLogAs: () => { throw new Error('never get here') },
   showUI: () => { throw new Error('never get here') },
   loadWebUIFromDist: serve({ directory: path.resolve(__dirname, '../renderer/dist') }),
-  confirmChangeWalletAddress: () => { throw new Error('never get here') }
+  confirmChangeWalletAddress: () => { throw new Error('never get here') },
+  restart: () => { throw new Error('never get here') }
```

Comment:
This is not a generic restart, the implementation calls `autoUpdater.quitAndInstall()` under the hood.

I propose renaming this new IPC method to `quitAndInstall` or something similar (`restartToUpdate`?) to better communicate that this API is for applying updates only.

What do you think?

Reply:
`restartToUpdate` sounds good

Reply:
good idea!